src/components/Mention.astro

```diff
@@ -1,12 +1,10 @@
 ---
-import { getAuthor } from '../data/authors';
-import Avatar from './Avatar.astro';
-const { name } = Astro.props;
+import { getAuthor } from '../data/authors'
+import Avatar from './Avatar.astro'
+const { name } = Astro.props
 
-const mention = await getAuthor(name);
-if (!mention) {
-    throw new Error(`Could not find mentioned user "${name}"!`)
-}
+const mention = getAuthor(name)
+if (mention instanceof Error) throw Error
```

Comment:
looks like this should be `throw mention`

Although, this looks the same as `getAuthor` throwing the error itself, but with extra steps :thinking: `getAuthor` doesn't even appear to return `Error`

Reply:
Ah yep sorry, used to return the error object, and it now throws internally. Good to remove this